### PR TITLE
Use predefined age aggregation patterns and scoring based on multiple places

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: server/tests/range_test.py [hunk 1]
```diff
@@ -18,53 +18,227 @@
...
+                'Count_Person_65To74Years': ['Count_Person_65To74Years'],
+                'Count_Person_75OrMoreYears': ['Count_Person_75OrMoreYears']
+            },
+            'geoId/12345': {
+                'Count_Person_5To17Years': ['Count_Person_5To17Years'],
+                'Count_Person_18To24Years': ['Count_Person_18To24Years'],
+                'Count_Person_25To34Years': ['Count_Person_25To34Years'],
+                'Count_Person_35To44Years': ['Count_Person_35To44Years'],
+                'Count_Person_45To54Years': ['Count_Person_45To54Years'],
+                'Count_Person_65To74Years': ['Count_Person_65To74Years']
```
Comment (beets) [inline server/tests/range_test.py:130]: this should also have 55-64

File: server/tests/range_test.py [hunk 2]
```diff
@@ -18,53 +18,227 @@
...
+            }
+        }
+        assert lib_range.aggregate_age_stat_var(input) == expected
+
+    def test_eu_place_age(self):
+        input = {
+            "country/USA": [
```
Comment (beets) [inline server/tests/range_test.py:137]: maybe update the dcid for this example so it's clearer?

File: server/tests/range_test.py [hunk 3]
```diff
@@ -18,53 +18,227 @@
...
+                'Count_Person_40To49Years': [
+                    'Count_Person_45To49Years', 'Count_Person_40To44Years'
+                ],
+                'Count_Person_50To59Years': [
+                    'Count_Person_55To59Years', 'Count_Person_50To54Years'
+                ],
+                'Count_Person_60To69Years': [
+                    'Count_Person_65To69Years', 'Count_Person_60To64Years'
+                ],
+                'Count_Person_70To79Years': [
```
Comment (beets) [inline server/tests/range_test.py:215]: ideally, we'd do 70+ in this case but it's ok for now.

**Review by beets — DISMISSED** [review]

File: server/tests/range_test.py (hunk 3 above)
Comment (shifucun) [reply server/tests/range_test.py]: Updated to use 70+

File: server/tests/range_test.py (hunk 2 above)
Comment (shifucun) [reply server/tests/range_test.py]: done

File: server/tests/range_test.py (hunk 1 above)
Comment (shifucun) [reply server/tests/range_test.py:135]: good catch!

**Review by beets — APPROVED** [review]

**Pull request merged**